### DISCUSSION TITLE
build.rs breaks cargo check?

### DISCUSSION THREAD

**Discussion by dgulotta** [slint-ui/slint#6909] [Q&A]
When running "cargo check" on a Slint project (using the build.rs file from the tutorial), it does not appear to check any Rust files besides src/main.rs.  I can put random garbage in src/lib.rs and it won't complain.  If I run "cargo build", it will find the errors in lib.rs.  Do I need to change some settings to get "cargo check" to look at lib.rs?

**Comment by ogoffart** [comment]
There should be a main.rs,  but i wonder why you have a lib.rs for an application (unless you are targetting wasm or android)
In this case, perhaps the lib.rs has a #[cfg] or something like that that makes it not be compiled with cargo check?
because for me it does try to compile a simple lib.rs  if i put one there.

**Comment by dgulotta** [comment]
Hmm, running "cargo clean" seems to have fixed the issue.  I guess it may have just been a random cargo bug.
